kura/examples/org.eclipse.kura.demo.modbus/OSGI-INF/metatype/org.eclipse.kura.demo.modbus.ModbusExample.xml

```diff
@@ -74,6 +114,63 @@
             required="true"
             default="1" 
             description="The modbus address of the device to be queried."/>
+            
+        <AD id="ipAddress"  
+            name="ipAddress"
+            type="String"
+            cardinality="0" 
+            required="false"
+            default="" 
+            description="IP address of Modbus server (TCP mode only)."/>
+
+         <AD id="tcp.port"  
+            name="tcp.port"
+            type="String"
```

Comment:
type should be integer with max 65535 and min 1 set - This would prevent users to provide alpha characters to the input field.